Core/Core.csproj

```diff
@@ -20,6 +20,7 @@
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
     <DefineConstants>TRACE;DEBUG;TypePipe;FEATURE_REMOTING;FEATURE_ASSEMBLYBUILDER_SAVE</DefineConstants>
+    <DefineConstants Condition="'$(TargetFrameworkVersion)' != '' OR '$(TargetFramework)' == 'net45'">$(DefineConstants);FEATURE_PDBEMIT</DefineConstants>
```

Comment:
we can keep this simple and add the feature to the constants like the rest. when we switch to cps style and then introduce .net 5, we introduce .net-dependent feature selection.